src/platform/utilities/ui/Breadcrumbs.jsx

```diff
@@ -0,0 +1,118 @@
+// TODO: Move to Jean Pants when approved
+import PropTypes from 'prop-types';
+import React from 'react';
+import _debounce from './debounce';
+import uniqid from 'uniqid';
+
+class Breadcrumbs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      mobileShow: false,
+    };
+
+    this.id = this.props.id || uniqid('va-breadcrumbs-');
+    this.listId = this.props.listId || uniqid('va-breadcrumbs-list-');
+    this.mobileWidth = this.props.mobileWidth || 425;
+
+    this.debouncedToggleDisplay = this.debouncedToggleDisplay.bind(this);
+    this.renderBreadcrumbLinks = this.renderBreadcrumbLinks.bind(this);
+    this.renderMobileLink = this.renderMobileLink.bind(this);
+    this.toggleDisplay = this.toggleDisplay.bind(this, this.listId);
+  }
+
+  componentDidMount() {
+    this.toggleDisplay();
+    window.addEventListener('resize', this.debouncedToggleDisplay);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.debouncedToggleDisplay);
+  }
+
+  debouncedToggleDisplay = _debounce(() => {
+    this.toggleDisplay();
+  }, 500);
+
+  toggleDisplay() {
+    if (window.innerWidth <= this.mobileWidth) {
+      this.setState({ mobileShow: true });
+    } else {
+      this.setState({ mobileShow: false });
+    }
+  }
+
+  renderBreadcrumbLinks() {
+    return React.Children.map(this.props.children, (child, i) => {
+      if (i === this.props.children.length - 1) {
+        return (
+          <li>{React.cloneElement(child, {
+            'aria-current': 'page',
+          })}</li>
+        );
+      }
+
+      return <li>{React.cloneElement(child)}</li>;
+    });
+  }
+
+  renderMobileLink() {
+    // The second to last link being sliced from the crumbs array
+    // prop to create the "Back by one" mobile breadcrumb link
+    return React.Children.map(this.props.children, (child, i) => {
+      if (i === this.props.children.length - 2) {
+        return (
+          <li>{React.cloneElement(child, {
+            'aria-label': `Previous page: ${child.props.children}`,
+            className: 'va-nav-breadcrumbs-list__mobile-link',
+          })}</li>
+        );
+      }
+
+      return null;
+    });
+  }
+
+  render() {
+    const {
+      ...breadcrumbProps
```

Comment:
I'm not sure I understand why this is being done.

Reply:
@jbalboni There's a few instances where custom props are being fed into the `<Breadcrumb>` component, like the Facility Locator. I added this `...breadcrumbProps` defensively to ensure those could be added as necessary. Will remove if it's not something we want here.

Reply:
Ok. It looks like you could `...this.props` later in this method to get the same effect without this line, unless I'm missing something.

Reply:
@jbalboni Some instances of the `<Breadcrumb>` have custom props passed down, like the Facility Locator. I'm not sure they're needed, but didn't want to remove them either. Added this as a defensive way to not break things. Can remove as needed.